apps/server/src/modules/oauth-provider/mapper/oauth-provider-request.mapper.spec.ts

```diff
@@ -1,31 +1,28 @@
-import { OauthProviderRequestMapper } from '@src/modules/oauth-provider/mapper/oauth-provider-request.mapper';
+import { AcceptLoginRequestBody } from '@shared/infra/oauth-provider/dto';
 import { LoginRequestBody } from '@src/modules/oauth-provider/controller/dto';
-import { AcceptLoginRequestBody } from '@shared/infra/oauth-provider/dto/index';
+import { OauthProviderRequestMapper } from '@src/modules/oauth-provider/mapper/oauth-provider-request.mapper';
```

Comment:
For the imports from inside the same module we want to use relativ pathes, but with index files if possible.